docs/pages/setup/reference/config.mdx

```diff
@@ -484,6 +484,12 @@ auth_service:
     # See "TLS Routing" in Architecture section for additional information.
     proxy_listener_mode: multiplex
 
+    # Tells tsh to load the host CAs of all clusters when trying to ssh into a node,
```

Comment:
If we're talking about Teleport SSH Service instances, we should say "Node" (or "Teleport Node", or "SSH Service instance") instead of "node" in this paragraph. This is to be consistent with the rest of our docs, and to make a distinction with hosts that aren't joined to a Teleport cluster.